### PR TITLE
Create Testing your Kubernetes Cluster - Acceptance and Conformance

### PR DESCRIPTION
**Pull request by chaithramohan86**
How to perform a simple acceptance testing on your freshly created cluster or existing cluster using kuberang and how to conformance test your k8s cluster using sonobuoy.

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by Rajakavitha1** [comment]
Thanks for the PR @chaithramohan86!!!! Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.

**Comment by chaithramohan86** [comment]
> Thanks for the PR @chaithramohan86!!!! Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.

hi @Rajakavitha1  I am contributing as an employee of Infosys and I get this message when i try to follow the process

"You are already authorized to contribute code to this project through your membership with CNCF - Infosys Ltd."

**Comment by Rajakavitha1** [comment]
> 
> 
> > Thanks for the PR @chaithramohan86!!!! Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> > Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.
> 
> hi @Rajakavitha1 I am contributing as an employee of Infosys and I get this message when i try to follow the process
> 
> "You are already authorized to contribute code to this project through your membership with CNCF - Infosys Ltd."

You are cla now reads yes!!! you are authorized to contribute. @bobsky will review your PR . Thanks!!!!

**Comment by chaithramohan86** [comment]
@bobsky kindly review the blog

**Comment by zparnold** [comment]
@kbarnard10 @zacharysarah Could I please have you guys take a look at this per chance?

File: content/en/blog/_posts/Testing your Kubernetes Cluster - Acceptance and Conformance [hunk 1]
```diff
@@ -0,0 +1,226 @@
+Testing the Kubernetes Cluster – Acceptance and Conformance.
```
Comment (zacharysarah) [inline content/en/blog/_posts/Testing your Kubernetes Cluster - Acceptance and Conformance]: This post needs front matter. For examples, take a look at other files in https://github.com/kubernetes/website/tree/master/content/en/blog/_posts.

**Comment by zacharysarah** [comment]
I'm deferring to @kbarnard10 for publishing.

**Comment by chaithramohan86** [comment]
Thanks @zacharysarah . Kindly do the needful 
@kbarnard10

**Comment by chaithramohan86** [comment]
hi @kbarnard10 kindly review

**Comment by zacharysarah** [comment]
@kbarnard10 👋 Ping

**Comment by kbarnard10** [comment]
@chaithramohan86 Sorry for the delay. This post is a little light on content and some of the versions are outdated. I'm going to close this for now. If you would like to expand and resubmit, please feel free to reopen or submit a Google Doc to k8sblog@linuxfoundation.org for review.

**Pull request closed without merging**